src/components/ListingOnCars.vue

```diff
@@ -104,59 +115,68 @@ export default class ListingOnCars extends Vue {
       cambio: "",
       tetoSolar: "",
       computadorDeBordo: "",
+      id: "",
     };
     this.isCreate = true;
     this.openFormModal();
   }
+  private checkIfYouFellOnGet() {
+    if (this.dataCars.length == 0) {
+      this.msgRequiredError = true;
+    } else {
+      this.msgRequiredError = false;
+    }
```

Comment:
daria pra fazer um if ternário 
```suggestion
this.msgRequiredError = this.dataCars.length == 0 ? true : false
    
```

Reply:
✔